sys/ztimer/Makefile.dep

```diff
@@ -8,6 +8,10 @@ ifneq (,$(filter ztimer,$(USEMODULE)))
   USEMODULE += ztimer_convert_frac
   USEMODULE += ztimer_convert_shift
 
+  # If ztimer is used enable rtt autoselection for ztimer_msec, and ztimer_sec.
+  # This will enable inclusion of `ztimer_periph_rtt` if `periph_rtt` is supported.
+  DEFAULT_MODULE += ztimer_periph_rtt_auto_select
```

Comment:
this line is not yet evaluated when l 96 is reached -> no auto selection is done